rcl/src/rcl/logging_rosout.c

```diff
@@ -438,20 +439,23 @@ rcl_logging_rosout_add_sublogger(
   rosout_map_entry_t entry;
   rosout_sublogger_entry_t sublogger_entry;
 
+  RCL_CHECK_ARGUMENT_FOR_NULL(logger_name, RCL_RET_INVALID_ARGUMENT);
+  RCL_CHECK_ARGUMENT_FOR_NULL(sublogger_name, RCL_RET_INVALID_ARGUMENT);
+  rcutils_ret_t rcutils_ret = rcutils_hash_map_get(&__logger_map, &logger_name, &entry);
+  if (RCL_RET_OK != (status = rcl_convert_rcutils_ret_to_rcl_ret(rcutils_ret))) {
+    if (RCL_RET_NOT_FOUND == status) {
+      RCL_SET_ERROR_MSG_WITH_FORMAT_STRING("Failed to get logger entry for '%s'.", logger_name);
+    }
+    return status;
```

Comment:
I think we can make this easier to read with:

```suggestion
  if (RCUTILS_RET_OK != rcutils_ret) {
    if (RCUTILS_RET_NOT_FOUND == status) {
      RCL_SET_ERROR_MSG_WITH_FORMAT_STRING("Failed to get logger entry for '%s'.", logger_name);
    }
    return rcl_convert_rcutils_ret_to_rcl_ret(rcutils_ret);
```